src/conductors/positive-batch-conductor.ts

```diff
@@ -0,0 +1,30 @@
+import { OASResult } from '../results';
+import PositiveConductor from './positive-conductor';
+import { Config } from '../config';
+import { FileIn } from '../utilities/file-in';
+
+export default class PositiveBatchConductor {
```

Comment:
I realize this new class follows up on previous work for batch testing of multiple APIs but I am wondering if the class structure is inverted.  

What happens when we have a set of different test run against multiple APIs.  Do we expect to have a batch class for all the different API test that then trigger this batch processor for positive.  I think a batch handling would be outside the scope of test run.

Reply:
Not sure I follow. There was a positive-all command added for [API-13164](https://vajira.max.gov/browse/API-13164). This is helpful at least for now so that the positive command can be run against multiple APIs at the same time while we have to kick them off manually. Once Docserver can run LOAST I agree it doesn't matter if Docserver would use the batch solution or not. But this is helpful for manual execution